test/e2e/tsdx-build-default.test.ts

```diff
@@ -51,6 +51,22 @@ describe('tsdx build :: zero-config defaults', () => {
     expect(output.code).toBe(0);
   });
 
+  it('should bundle regeneratorRuntime', () => {
+    const output = execWithCache('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+
+    const matched = grep(/regeneratorRuntime = r/, ['dist/build-default.*.js']);
+    expect(matched).toBeTruthy();
+  });
+
+  it('should not bundle regeneratorRuntime when targeting Node', () => {
+    const output = execWithCache('node ../dist/index.js build --target node');
```

Comment:
woops, `build-default` isn't supposed to include options.. must've forgot how I setup these tests 😕

Reply:
`build-options` was only _just_ added with my smoke test in #896, so I must've forgot to split out files before. #903 fixes this